app/views/register-with-a-gp/donor-organs.html

```diff
@@ -0,0 +1,92 @@
+{% extends 'templates/nhs_transaction_layout.html' %}
+{% import 'includes/form_macros.html' as form_macros %}
+
+{% block afterHeader %}
+  {% include "includes/service-headers/register-with-a-gp.html" %}
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+  <h1 class="heading-xlarge">
+    Organ donation
+  </h1>
+
+  <form action="" method="get" class="form text">
+
+    <fieldset class="form-group">
+      <legend class="form-label-bold">Are you willing to donate your organs for transplantation after your death?</legend>
+      <input type="radio" id="organs-yes" name="organs">
+      <label class="form-control-radio js-toggle" for="organs-yes" data-show="#organ-donor">
+        Yes
+      </label>
+      <input type="radio" id="organs-no" name="organs">
+      <label class="form-control-radio js-toggle" for="organs-no" data-hide="#organ-donor">
+        No
+      </label>
+    </fieldset>
+
+    <div class="js-hidden panel-indent" id="organ-donor">
+      <fieldset>
+        <legend>I will donate</legend>
+
+        <div class="form-group">
+          <input type="checkbox" id="any-part-of-my-body" checked="checked">
```

Comment:
@evilstreak there's probably more thinking involved here... for the time being let's roll with this. But we need to get to a point where the user is saying either:

"I'm happy to donate any part of my body"
OR
"I am ONLY happy for you to use the following organs"

...which basically seems odd. We should find out the reasoning behind this.